docs/markdown/snippets/pkg-config-fix-static-only.md

```diff
@@ -0,0 +1,12 @@
+## Improved generation of pkg-config files for static only libraries.
+
+Previously pkg-config files generated by the pkgconfig modules for static libraries
+with dependencies could only be used in a dependencies with `static: true`.
+
+Now the generated file contains the needed dependencies libraries directly within
+`Requires` and `Libs` for build static libraries passed via the `libraries` keyword
+argument.
+
+Projects that install both a static and a shared version of a library should always
+pass the shared library to generate to create a setup where users can link to the
```

Comment:
s/to generate // ?

Reply:
I'm rewriting the last paragraph anyway, because just passing the shared library doesn't work anymore. Now the user either needs both_libraries (when merged) or has to manually specify dependencies.